package/src/components/Channel/hooks/useCreatePaginatedMessageListContext.ts

```diff
@@ -17,15 +18,9 @@ export const useCreatePaginatedMessageListContext = <
   setLoadingMoreRecent,
 }: PaginatedMessageListContextValue<StreamChatGenerics> & {
   channelId?: string;
-}) => {
-  const messagesUpdated = messages
-    .map(
-      ({ deleted_at, latest_reactions, reply_count, status, updated_at }) =>
-        `${deleted_at}${
-          latest_reactions ? latest_reactions.map(({ type }) => type).join() : ''
-        }${reply_count}${status}${updated_at?.toISOString?.() || updated_at}`,
-    )
-    .join();
+  }) => {
+  
+  const messagesUpdated = messages.map((props) => mapMessages(props)).join();
```

Comment:
`messages.map(mapMessages).join();` this should be enough

Reply:
✅